fdm-app/app/components/blocks/measures/add-measure-dialog.tsx

```
@@ -0,0 +1,583 @@
+    // Close dialog and reset on successful submission
+    const isSubmitting = fetcher.state !== "idle"
+    const prevState = useRef(fetcher.state)
+    useEffect(() => {
+        if (prevState.current !== "idle" && fetcher.state === "idle") {
+            onOpenChange(false)
+        }
+        prevState.current = fetcher.state
+    }, [fetcher.state, onOpenChange])
+
+    // Reset form when dialog opens
+    useEffect(() => {
```

Biome (2.4.15)

[error] This hook does not specify its dependency on form.reset.

(lint/correctness/useExhaustiveDependencies)